calculators/gas_king_rating

```diff
@@ -0,0 +1,101 @@
+<?php
+
+$GLOBALS['rating_calculators']['gas_king_rating'] = [
+    'description' => '
+        gas king rating
+        only objective: gas as much and as soon as you can
+
+        you start with 2000 rating (compression is activated although maybe not efficient, I was too lazy to see how you did it)
+        you lose 10 rating per minute for each gas not taken
+        you win 1, 2, 3, 4 rating per minute for gas 1, 2, 3, 4
+        ',
+    'default_rating' => 2000,
+    'fields' => [
+        'g.duration',
+        'gp.gas_timings',
+        'gp.gas_count',
+        ],
+    'main' => 'dsr_gas_king_rating__main',
+    ];
+
+function same_sign($a, $b)
```

Comment:
name of every function you write must have same prefix

Reply:
I think I fixed it, although I tried to understand what was a prefix but didn't quite understand

Reply:
prefix is always _name_of_rating__function_name(), in your case it should be _mignoubou_gas_king__main()
in case of builtin ratings (not custom rating like the one you are making) I also prepend dsr_